blackjax/kernels.py

```diff
@@ -899,7 +894,7 @@ def run(rng_key: PRNGKey, positions: PyTree, num_steps: int = 1000):
     return AdaptationAlgorithm(run)  # type: ignore[arg-type]
 
 
-class rmh:
+class random_walk:
```

Comment:
No need to implement this as it is a very basic example of random walk metropolis hastings that the user could easily implement with a more general `random_walk` kernel (see below).